requirements.txt

```diff
@@ -0,0 +1,2 @@
+xml2rfc==3.12.10
```

Comment:
Hmm, does this really need to be tied to a specific version?  I know it is good practice, but it's a pain in the butt.

Reply:
Also, if we don't include the version, it's going to make CI cache file updating problematic

Reply:
Perhaps what we can do is list `xml2rfc>=3.12.10` (and similar for other deps), so the latest will be installed, but we can still bump it up once in a while to assure that old installations get upgraded.